src/reporting/schemas/method_info_schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openproblems.bio/schemas/method_info_schema.json",
+  "title": "Method info",
+  "description": "Information about methods in a task",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "description": "Method identifier",
+        "type": "string"
+      },
+      "label": {
+        "description": "Human-readable label for the method",
+        "type": "string"
+      },
+      "commit": {
+        "description": "Git commit SHA for this version of the method",
+        "type": "string"
+      },
+      "summary": {
+        "description": "One line summary of the method",
+        "type": "string"
+      },
+      "description": {
+        "description": "Detailed description of the method",
+        "type": "string"
+      },
+      "type": {
+        "description": "Type of the method. Either 'control_method' (a positive or negative control used for scaling metric values), 'method' (a real method that is being benchmarked) or 'reference_method' (another method that is included in results for comparison but not used for scaling)",
+        "type": "string",
+        "enum": ["control_method", "method", "reference_method"]
+      },
+      "link_code": {
+        "description": "URL to the package or code provided by the method authors",
+        "type": "string"
+      },
+      "link_documentation": {
+        "description": "URL to documentation provided by the method authors",
+        "type": ["null", "string"]
+      },
+      "link_implementation": {
+        "description": "Permalink URL to the version of the method component used",
+        "type": "string"
+      },
+      "link_container_image": {
+        "description": "URL to the container image used by the method component",
+        "type": "string"
+      },
+      "references": {
+        "$ref": "https://openproblems.bio/schemas/references_schema.json"
+      },
+      "additional_info": {
+        "description": "Additional information from the method config",
+        "type": "object"
+      },
+      "version": {
+        "description": "Tag for this version of the task",
+        "type": "string"
+      }
+    },
```

Comment:
maybe it'd be better to add authors to datasets, methods, and metrics as well; in case somebody is not / does not want to be a task contributor but would like to contribute datasets / methods / metrics.

Reply:
We could add fields for them I guess but they would be empty until that information is in the configs.

For datasets that might make sense but I feel like a method/metric contribution is still a contribution to the task. Unless you want to know who worked on specific components?

Reply:
> We could add fields for them I guess but they would be empty until that information is in the configs.

They would be, but at least the website DB would already have the necessary fields

> For datasets that might make sense but I feel like a method/metric contribution is still a contribution to the task. Unless you want to know who worked on specific components?

Well maybe somebody wants to contribute a method but not claim to be involved in the benchmark -- E.g. they added their own method to the benchmark, but they don't want to be an author / contributor to the benchmark to avoid conflict of interest